src/xrServerEntities/script_ini_file_script.cpp

```diff
@@ -101,7 +102,10 @@ static void CScriptIniFile_Export(lua_State* luaState)
             .def("remove_line", &CScriptIniFile::remove_line)
             .def("set_override_names", &CScriptIniFile::set_override_names)
             .def("section_count", &CScriptIniFile::section_count)
+            .def("section_for_each", &CScriptIniFile::section_for_each)
+            .def("set_readonly", &CScriptIniFile::set_readonly)
             //Alundaio: END
+            .def("fname", REMOVE_NOEXCEPT(&CScriptIniFile::fname))
```

Comment:
Looks like wrong thing to do and we should remove `noexcept` specifier *literally*. (comment it out in `fname` function)

Reply:
I just removed `REMOVE_NOEXCEPT`.

Reply:
The `fname` function has `noexcept` specifier, but it's not prevent build fail

Reply:
Yes, the compilation fails exactly because of that. `noexcept` should be removed from the function itself. (commented out)